lib/matplotlib/axes/_axes.py

```diff
@@ -5037,23 +5039,25 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
 
         Parameters
         ----------
-        X : array_like, shape (n, m) or (n, m, 3) or (n, m, 4)
+        X : array_like, shape (n, m) or (n, m, 3) or (n, m, 4) or (2, n, m)
             Display the image in `X` to current axes.  `X` may be an
             array or a PIL image. If `X` is an array, it
             can have the following shapes and types:
 
-            - MxN -- values to be mapped (float or int)
+            - MxN -- univariate values to be mapped (float or int)
             - MxNx3 -- RGB (float or uint8)
             - MxNx4 -- RGBA (float or uint8)
+            - 2xMxN -- bivariate values to be mapped (float or int)
```

Comment:
What happens when the input is shape `(2, 3, 4)`?

This sort of ambiguity should be avoided (and indicates that this API is probably not the right design choice).